src/Microsoft.Health.Dicom.Api/Features/Converter/CustomJsonStringEnumConverter.cs

```diff
@@ -0,0 +1,33 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+
+
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using EnsureThat;
+
+namespace Microsoft.Health.Dicom.Api.Features.Converter
+{
+    internal class CustomJsonStringEnumConverter : JsonConverterFactory
+    {
+        public CustomJsonStringEnumConverter() { }
+
+        public override bool CanConvert(Type typeToConvert)
+        {
+            EnsureArg.IsNotNull(typeToConvert, nameof(typeToConvert));
```

Comment:
nit: you can do this in one line like
```suggestion
            => EnsureArg.IsNotNull(typeToConvert, nameof(typeToConvert)).IsEnum;
```